biofefi/machine_learning/learner.py

```diff
@@ -433,6 +455,62 @@ def _evaluate(
     return eval_res
 
 
+def calculate_regression_metrics(y_true: np.array, y_pred: np.array, metric_function):
```

Comment:
If this function is only used in this module, you can add an underscore to the front of the name to indicate it's private. That's the convention in python.